src/wp-includes/post.php

```diff
@@ -716,13 +716,23 @@ function create_initial_post_types() {
  *
  * @since 2.0.0
  *
- * @param int  $attachment_id Attachment ID.
- * @param bool $unfiltered    Optional. Whether to apply filters. Default false.
+ * @param int    $attachment_id Attachment ID.
+ * @param bool   $unfiltered    Optional. Whether to apply filters. Default false.
+ * @param string $unfiltered    Optional. The file MIME type to return.
  * @return string|false The file path to where the attached file should be, false otherwise.
  */
-function get_attached_file( $attachment_id, $unfiltered = false ) {
+function get_attached_file( $attachment_id, $unfiltered = false, $mime_type = null ) {
```

Comment:
After giving this another thought, I don't think we should add the `$mime_type` parameter to this function. The parameter's purpose right now really revolves around images, however this function is for _any_ attachment, not only images.

I think we should revert the changes in this function, also given that having the parameter here isn't crucial for anything else in this PR.